common/customizable_localization/wc_lifestyles_custom_loc.txt

```diff
@@ -2,48 +2,48 @@
 	type = character
```

Comment:
Characters that have historical perks set shouldn't get perks on-game start.
Like Antonidas starts the game learning Light magic.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/155839094-311d278b-d41c-4f8d-a3ee-dedca660d3b7.png)

</details>

Reply:
Implemented in 890808f55